.github/actions/docker-scan/action.yml

```diff
@@ -32,6 +32,12 @@ runs:
         sed -i 's/pkg:apk/pkg:alpine/g' dependency-results.sbom.json
       shell: bash
 
+    - name: replace correlator
+      run: |
+        jq '.job.correlator = "trivy_'${{ inputs.docker_image }}'"' dependency-results.sbom.json > sbom.tmp \ 
```

Comment:
I've been thinking about this more and I think we should pass in a dedicated input for this correlator.  Something like `${{ inputs.sbom_name }}`.  

Reason being is that our staging and production docker images will have different names, which will result in 2 dependency snapshots for the same image.

Reply:
done